features/xacml/org.wso2.carbon.identity.xacml.server.feature/resources/entitlement.properties

```diff
@@ -26,7 +26,7 @@ PDP.AttributeCaching.CachingInterval=300
 PDP.ResourceCaching.Enable=true 
 PDP.ResourceCaching.CachingInterval=300
 PDP.SchemaValidation.Enable=true
-PDP.Balana.Config.Enable=false
+PDP.Balana.Config.Enable=true
```

Comment:
@omindu "PDP.Balana.Config.Enable" attribute is read at the time of entitlement component initialization. Since we have to use a custom functionId Balana should initialize from balana-config.xml. Making this attribute true will allow the entitlement component to read from balana-config.xml instead of its default config file.